extensions/wikia/Search/classes/Test/ResultTest.php

```diff
@@ -90,36 +92,36 @@ public function testGetTextWithWordLimit() {
 	/**
 	 * @group Slow
 	 * @slowExecutionTime 0.10056 ms
-	 * @covers Wikia\Search\Result::getEscapedUrl
+	 * @covers \Wikia\Search\Result::getEscapedUrl
 	 */
 	public function testGetEscapedUrl() {
 		$mock = $this->getMockBuilder( 'Wikia\Search\Result' )
 			->disableOriginalConstructor()
-			->setMethods( ['getUrl'] )
+			->setMethods( [ 'getUrl' ] )
 			->getMock();
 
-		$mock->expects( $this->once() )
-			->method( 'getUrl' )
-			->will( $this->returnValue( 'test&amp;this_is_"' ) );
+		$mock->expects( $this->once() )->method( 'getUrl' )->will( $this->returnValue( 'test&amp;this_is_"' ) );
 
-		$this->assertEquals( 'test&amp;this_is_&quot;',
-			$mock->getEscapedUrl() );
+		$this->assertEquals(
+			'test&amp;this_is_&quot;',
+			$mock->getEscapedUrl()
+		);
 	}
 
 	/**
 	 * @group Slow
 	 * @slowExecutionTime 0.10045 ms
-	 * @covers Wikia\Search\Result::getTitle
-	 * @covers Wikia\Search\Result::setTitle
+	 * @covers \Wikia\Search\Result::getTitle
+	 * @covers \Wikia\Search\Result::setTitle
 	 */
 	public function testTitleFieldMethods() {
 
 		global $wgLanguageCode;
 		$wgLanguageCode = 'fr';
 
 		$fieldsCopy = $this->defaultFields;
-		unset($fieldsCopy['title']);
-		unset($fieldsCopy[Utilities::field('title')]);
+		unset( $fieldsCopy['title'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)